packages/views.api.unstable/src/manifest.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  EventDefinition,
+  ParameterConfig,
+  ParameterDefinition,
+} from "./config.js";
+
+export interface ViewManifest {
+  views: Record<string, ViewManifestConfig>;
+}
+
+export interface ViewManifestConfig {
+  rid: string;
+  version: string;
+  entrypointJs: string[];
+  entrypointCss?: string[];
+  parameters: Record<string, ParameterDefinition>;
+  events: Record<string, EventDefinition<ParameterConfig>>;
+}
+
+export const MANIFEST_FILE_LOCATION = ".palantir/view-config.json";
```

Comment:
Should we follow naming convention of `foundry.config.json`? And make it plural given that it can publish multiple views
```suggestion
export const MANIFEST_FILE_LOCATION = ".palantir/views.config.json";
```

Reply:
oh fair, i can rename. no strong opinion on `-` vs `.` so i'll just match `foundry.config.json`

I think since this lives on the client side vs literally in the foundry system, i kind of prefer `.palantir`